examples/InteractiveExample.tsx

```diff
@@ -212,10 +195,9 @@ export default class InteractiveExample extends React.Component<Props, State> {
       // fall through
       case PickingType.LOCATION_AREA: {
         if (object) {
-          const nextSelectedId = getLocationId(object);
           this.setState(state => ({
             ...state,
-            selectedLocationIds: getNextSelectedLocationIds(state.selectedLocationIds, nextSelectedId),
+            selectedLocationIds: [this.props.getLocationId!(object)],
```

Comment:
The usage of `!` can result to a crash since there are no defaultProps specified for this component.